tests/test_dataset_wrapper.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.model_selection import train_test_split
+
+from datasets.dataset_wrapper import DatasetWrapper
+
+wine_data_path = "../datasets/winequality.csv"
+random_state, test_size = 0, 0.2
+
+# test invalid initailization of DatasetWrapper object
+def test_fie_does_not_exist():
+    with pytest.raises(FileNotFoundError):
+        DatasetWrapper("datasets/winequalit.csv")
+
+def test_fie_format_incorrect():
+    with pytest.raises(IOError):
+        DatasetWrapper("datasets/README.md")
```

Comment:
I think this is testing the failure when you pass in a file that is not a CSV?

For me, this is actually __not__ failing, because Pandas is trying to be smart and is parsing the table we have in that README. If you're not getting this behaviour, please make sure you've activated the conda environment before running the tests.

On the other hand, I tried running this with a different file and it's giving a Pandas `ParserError` rather than `IOError`.

Reply:
I passed in a python script just to make this test fail. Not sure if we need to handle files like markdown as errors? I

Reply:
Using the setup.py script is fine. Anything that is not CSV should raise the error. What I meant was the markdown should have raised the exception as well, and I think that's what you intended, but it didn't because of a Pandas feature trying to recognize that the markdown in fact contains a table.